agent/engine/docker_task_engine.go

```diff
@@ -562,9 +563,17 @@ func (engine *DockerTaskEngine) createContainer(task *api.Task, container *api.C
 	// we have to do this in create, not start, because docker no longer handles
```

Comment:
Can you move this comment about HostConfig above line 579 instead of here?  It's less clear what the comment is applying to now.